tailwind.config.js

```diff
@@ -6,7 +6,11 @@ export default {
     './packages/lib/src/**/*.{svelte,js,ts,jsx,tsx}'
   ],
   theme: {
-    extend: {}
+    extend: {
+      colors: {
+        main: 'hsl(var(--s-main-color))'
```

Comment:
Setting it like this means we can use `text-main` `bg-main` `bg-main/50` etc

Reply:
Nice!